scalability_and_performance/low_latency_tuning/cnf-tuning-low-latency-nodes-with-perf-profile.adoc

```diff
@@ -0,0 +1,98 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cnf-tuning-low-latency-nodes-with-perf-profile"]
+= Tuning nodes for low latency with the performance profile
+include::_attributes/common-attributes.adoc[]
+:context: cnf-low-latency-perf-profile
+
+toc::[]
+
+Tune nodes for low latency by using the cluster performance profile.
+You can restrict CPUs for infra and application containers, configure huge pages and Hyper-Threading, and configure CPU partitions latency-sensitive processes.
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../scalability_and_performance/low_latency_tuning/cnf-provisioning-low-latency-workloads.adoc#cnf-provisioning-low-latency-workloads[Provisioning real-time and low latency workloads]
+
+[id="cnf-create-performance-profiles"]
+== Creating a performance profile
+
+Learn about the Performance Profile Creator (PPC) and how you can use it to create a performance profile.
+
+[NOTE]
+====
+Currently, disabling CPU load balancing is not supported by cgroup v2. As a result, you might not get the desired behavior from performance profiles if you have cgroup v2 enabled. Enabling cgroup v2 is not recommended if you are using performance profiles.
+====
```

Comment:
As mentioned in another comment I think this has been addressed, but maybe this isn't the time to remove it?

Reply:
Not in this PR, but we need to watch out for merge conflicts.